src/Form/RefillStationType.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\RefillStation;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+
+class RefillStationType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('name')
+            ->add('description', TextareaType::class)
+            ->add('debit')
+            ->add('installation')
+            ->add('refillTime')
+            ->add('additionalStorage')
+            ->add('photo')
```

Comment:
You should specify the class of all your field types to avoid surprises with the type guesser